demo/api/schema.gql

```diff
@@ -622,7 +622,7 @@ type Query {
   pageTreeNodeList(scope: PageTreeNodeScopeInput!, category: String): [PageTreeNode!]!
   paginatedPageTreeNodes(scope: PageTreeNodeScopeInput!, category: String, sort: [PageTreeNodeSort!], offset: Int! = 0, limit: Int! = 25): PaginatedPageTreeNodes!
   pageTreeNodeSlugAvailable(scope: PageTreeNodeScopeInput!, parentId: ID, slug: String!): SlugAvailability!
-  getSitePreviewJwt(path: String!, previewData: PreviewData!): String!
+  validateSitePreview(scope: JSONObject!): Boolean!
```

Comment:
naming: this does not validate a site preview (as it's name suggests); it returns true/false if site preview is allowed. Maybe `isSitePreviewAllowed` or `allowSitePreview`?